postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -386,12 +386,25 @@ def _get_db_explain_setup_state(self, dbname):
         try:
             result = self._run_explain(dbname, EXPLAIN_VALIDATION_QUERY, EXPLAIN_VALIDATION_QUERY)
         except psycopg2.errors.InvalidSchemaName as e:
-            self._log.warning("cannot collect execution plans due to invalid schema in dbname=%s: %s", dbname, repr(e))
+            self._check.warning(
+                "Unable to collect execution plans due to invalid schema in database '%s'. This could be due to a "
+                "missing 'datadog' schema in the database. See https://docs.datadoghq.com/database_monitoring/"
```

Comment:
> This could be due to a missing 'datadog' schema in the database

I am really afraid to insert debugging information like this directly into the Agent where we do not control the upgrade cycle at all. The setup can change quite a bit and we may no longer require datadog schema in the future (as an example). We may also discover new ways in which this error could arise that we document later.

Reply:
Yeah, I wasn't sure about this one. I had included it because I did get that error in this context and that would have been the solution ...but I had the same thought and doubts. 

I think it might make sense to keep the focus very narrow and only report warnings on the errors that we can have reliable and robust reporting on. Concretely, I think I'd just revert this one case back to what it was and skip emitting a warning.